content/en/docs/concepts/signals/logs.md

```diff
@@ -4,13 +4,102 @@ weight: 3
 ---
 
 A **log** is a timestamped text record, either structured (recommended) or
-unstructured, with metadata. While logs are an independent data source, they may
-also be attached to spans. In OpenTelemetry, any data that is not part of a
-distributed trace or a metric is a log. For example, _events_ are a specific
-type of log. Logs are often used to determine the root cause of an issue and
-typically contain information about who changed what as well as the result of
-the change.
-
-> For more information, see the [logs specification][].
+unstructured, with metadata. Of all telemetry signals logs have probably the
+biggest legacy. Most programming languages have built-in logging capabilities or
+well-known, widely used logging libraries. While logs are an independent data
+source, they may also be attached to spans. In OpenTelemetry, any data that is
+not part of a distributed trace or a metric is a log. For example, _events_ are
+a specific type of log. Logs are often used to determine the root cause of an
+issue and typically contain information about who changed what as well as the
+result of the change.
+
+For metrics and traces OpenTelemetry takes the approach of a clean-sheet design,
+specifies a new API and provides full implementations of this API in multiple
+languages.
+
+Our approach with logs is somewhat different. For OpenTelemetry to be successful
+in logging space we need to support existing legacy of logs and logging
+libraries, while offering improvements and better integration with the rest of
+observability world where possible.
+
+This is in essence the philosophy behind OpenTelemetry’s logs support. We
+embrace existing logging solutions and make sure OpenTelemetry works nicely with
+existing logging libraries, log collection and processing solutions.
```

Comment:
```suggestion
For traces and metrics, OpenTelemetry takes the approach of a clean-sheet design,
specifies a new API, and provides full implementations of this API in multiple
language SDKs.

OpenTelemetry's approach with logs is different. Because existing logging solutions
are widespread in language and operational ecosystems, OpenTelemetry acts as a
"bridge" between those logs, the tracing and metrics signals, and other OpenTelemetry
components. In fact, the API for logs is called the "Logs Bridge API" for this reason.
```